### PR TITLE
taxonomy: Taxonomy update (mostly Polish) - 2

### PR DESCRIPTION
**Pull request by stephanegigandet**
I will try to fix merge conflicts and tests for @ArturLange 's PR https://github.com/openfoodfacts/openfoodfacts-server/pull/8663

### CODE REVIEW

**Comment by ArturLange** [comment]
Thank you!
Unfortunately I was unable to make tests work

File: taxonomies/ingredients_processing.txt [hunk 1]
```diff
@@ -1282,7 +1302,7 @@ it:pastorizzato, pastorizzata, pastorizzate, pastorizzati
 ja:パスチャライズ
 nb:pasteurisert
 nl:gepasteuriseerd, gepasteuriseerde
-pl:pasteryzowana, pasteryzowane, pasteryzowany
+pl:pasteryzowana, pasteryzowane, pasteryzowany, pasteryzowanego, pasteryz<owanej
```
Comment (benbenben2) [inline taxonomies/ingredients_processing.txt:1305]: Is there really a "<" symbol in the middle of the word?

**Review by benbenben2 — APPROVED** [review]
I approve already, but look at the comment.

File: taxonomies/nutrient_levels.txt [hunk 2]
```diff
@@ -126,7 +126,7 @@ ny:Fat in low quantity
 oc:Fat dins low quantity
 oj:Fat in low quantity
 om:Fat in low quantity
-or:Fat in low quantity
+or:Fat ରେ low quantity
```
[the remark this answers is not in the document]
Comment (stephanegigandet) [reply taxonomies/nutrient_levels.txt:129]: That file is generated, I guess someone just translated the word "or", but not "fat" and "in low quantity"..

**Comment by benbenben2** [comment]
Kudos to both of you @stephanegigandet  and @ArturLange :+1:

**Pull request merged**